perlock-core/src/main/java/name/mitterdorfer/perlock/PathWatcher.java

```diff
@@ -17,12 +17,12 @@
      * @throws IllegalStateException If <code>#start()</code> is invoked on an already running
      *                               <code>PathWatcher</code>.
      */
-    void start() throws IOException;
+    PathWatcher start() throws IOException;
 
     /**
      * Determines whether this <code>PathWatcher</code> instance is currently running.
      *
-     * @return true iff this <code>PathWatcher</code> instance is currently running.
+     * @return true if this <code>PathWatcher</code> instance is currently running.
```

Comment:
the "iff" was intentional. It means "if and only if" and is quite common (see [Wikipedia](https://en.wikipedia.org/wiki/If_and_only_if)))